scripts/deploy.js

```diff
@@ -0,0 +1,53 @@
+const zipFolder = require('zip-folder');
+const fs = require('fs');
+
+let folderName = '../build/';
+let zipName = '../release/hypercrx.zip';
+
+// getting all the credentials and IDs from environment
+let REFRESH_TOKEN = process.env.REFRESH_TOKEN;
+let EXTENSION_ID = process.env.EXTENSION_ID;
+let CLIENT_SECRET = process.env.CLIENT_SECRET;
+let CLIENT_ID = process.env.CLIENT_ID;
+
+const webStore = require('chrome-webstore-upload')({
+  extensionId: EXTENSION_ID,
+  clientId: CLIENT_ID,
+  refreshToken: REFRESH_TOKEN,
+  clientSecret: CLIENT_SECRET,
+});
```

Comment:
I cannot find 'chrome-webstore-upload' and 'zip-folder' in `package.json`. 

If you can run it in your machine, there is chance that your `node_modules` folder still contains both thease packages since you didn't run `yarn install` after removing packges from `package.json`.